src/input/keys_constants.h

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#ifndef KEYS_CONSTANTS_H
+#define KEYS_CONSTANTS_H
```

Comment:
```suggestion
#pragma once
```
Always use pragma once, ditto for other headers